src/main.cpp

```diff
@@ -90,7 +90,7 @@ int main(int argc, char **argv)
                     // Click X on window
                     running = false;
                     break;
-                case SDL_KEYDOWN: [[fallthrough]]
+                case SDL_KEYDOWN: [[fallthrough]];
```

Comment:
This is not the only [[fallthrough]] statement in the codebase. Is only this one broken on Mac?

Reply:
Yeah, all the other fallthroughs I took a look at have semicolons after actually (though I didn't look at all of them).. not sure how it was compiling without it